client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line no-unused-vars
+import React, { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+import './Breadcrumbs.css'
+
+// eslint-disable-next-line react/prop-types
+const Breadcrumbs = ({ onClick }) => {
+  const [breadcrumbs, setBreadcrumbs] = useState([])
+  const location = useLocation()
+
+  useEffect(() => {
+    const pathnames = location.pathname.split('/').filter((x) => x)
+    let currentPath = '/'

```

Comment:
not the best idea to have currentPath always as "/" 
it changes, whenever you navigate to a different folder.
name `rootPath` is better for "/", but we don't really need it